org_chart.js

```diff
@@ -4,70 +4,42 @@ let elon = {
   position: "Boss",
 }
```

Comment:
What about elon?  He should be constructed too.
In this small example, there is only one manager, but what about when we extend this to larger org-charts?